.github/workflows/build-rust-code.yml

```diff
@@ -0,0 +1,101 @@
+name: Build Production Rust Code
```

Comment:
This runs on tests too right? Should "Production" be removed?

Reply:
it is only production.  tarpaulin and tests build differently.